### DISCUSSION TITLE
Latest version that runs on Nook Glow Light 3 (android 4.4.2)

### DISCUSSION THREAD

**Discussion by sidarous** [koreader/koreader#10102] [Q&A]
Hi community,

Any Nook users know the latest version that works on the Nook Glowlight 3? At some point, I think support for older versions of Android died. This makes complete sense and is likely necessary for KOReader to continue advancing. However, I want to make sure I use the latest possible version that is still compatible for my device. 

Long term, I'd like to get a Boox device but I'd like to get a year or two more from my Nook Glowlight 3, which has been quite nice and reliable for the 3 or so years I've been using it.

**Comment by mergen3107** [comment]
I think all version should work up to now?

**Reply by Frenzie** [reply]
Not all, 2.3 is no longer supported since <https://github.com/koreader/android-luajit-launcher/pull/91>. But the Nook Glowlight 3 has 4.4.x according to a quick search, so that should work perfectly fine.

**Comment by sidarous** [comment]
Okay, I tested all the recent releases and found out it's just the most recent that is not working. I will investigate further. Thanks.

**Reply by gk58tbkkfy-boop** [reply]
Hi!  I know it’s been over three year, but I was wondering if you know if this still works?  I got a Nook Glowlight 3 Plus off FB marketplace a few weeks ago.  It works fine, but I wanted to see if I could sync it up to another, smaller, reader if I got one. I’ve looked into rooting it and it seems complicated.  Much more complicated than the newer devices. Honestly, I don’t wanna try it so if I’d have to read it to put  KoReader on it, that’s gonna be a nope. But I saw somewhere that you didn’t have to root. The information on the glow light three is confusing. Then again it’s really old, so understandable.

**Reply by Frenzie** [reply]
Just try the latest version? It'll only take a few seconds.
